modules/spark-wallet.nix

```diff
@@ -63,11 +64,9 @@ in {
         User = "clightning";
         Restart = "on-failure";
         RestartSec = "10s";
-        PrivateTmp = "true";
-        ProtectSystem = "full";
-        NoNewPrivileges = "true";
-        PrivateDevices = "true";
-      };
+      } // nix-bitcoin-services.defaultHardening
+        // nix-bitcoin-services.node
+        // nix-bitcoin-services.allowTor;
```

Comment:
Should this prevent spark-wallet from reaching bitstamp.net for USD price query?

Reply:
did you test that? I was under the impression that this is handled client side.

Reply:
Not seeing price in mobile app and getting this error for spark-wallet service
```
Apr 28 18:42:05 nix-bitcoin bash[2085]: Error: getaddrinfo EBUSY www.bitstamp.net www.bitstamp.net:443
Apr 28 18:42:05 nix-bitcoin bash[2085]:     at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:56:26)
```